container/common/python3.yaml

```diff
@@ -15,4 +15,4 @@ commandTests:
 - name: 'python3-version'
```

Comment:
We can add a new command test to test add_apt_repository

Reply:
done